v0/destinations/adobe_analytics/transform.js

```diff
@@ -439,6 +439,9 @@ const process = async event => {
     case EventType.TRACK:
     case EventType.PAGE:
     case EventType.SCREEN:
+      if (messageType === "page" || messageType === "screen") {
+        message.event = message.name;
```

Comment:
Unless we need this updated message object in Ln 452, we should avoid updating the input.
```suggestion
        const messageClone = { ...message };
        messageClone.event = message.name;
```

Reply:
addressed